src/checkers/inference/InferenceMain.java

```diff
@@ -453,6 +453,9 @@ public DefaultResultHandler(Logger logger) {
 
         @Override
         public void handleCompilerResult(boolean success, String javacOutStr) {
+            if (!javacOutStr.isEmpty()) {
```

Comment:
Think of an appropriate way to report `AlwaysFalseConstraint` in INFER mode, similar to TYPE_CHECK mode.